app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -160,6 +170,14 @@ export default function createRPCMethodTrackingMiddleware({
         properties.method = method;
       }
 
+      if (method === MESSAGE_TYPE.PERSONAL_SIGN) {
+        const data = req && req.params && req.params[0];
+        const { isSIWEMessage } = detectSIWE({ data });
+        if (process.env.SIWE_V1 && isSIWEMessage) {
```

Comment:
[nit] I think we can move the `process.env.SIWE_V1` check into the if statement above, `if (process.env.SIWE_V1 && method === MESSAGE_TYPE.PERSONAL_SIGN) {`

same comment in the next callback below, line 216

Reply:
Done!